### ISSUE TITLE
Fellow ___metrician

### ISSUE THREAD

**Issue by caseykneale** [Nosferican/Econometrics.jl#23]
Hey there,

Noticed your package. I like your code style. I am running a package: https://github.com/caseykneale/ChemometricsTools.jl and it has similar elements.

I figured I'd reach out and say hi. Maybe there's a way we can abstract our works into a larger "JuliaMetrics" landscape or something?

**Comment by Nosferican** [comment]
Hi! I am all for nice abstraction and improving the Julia ecosystem. This package builds on a few abstractions:
- Statistical Model abstraction from (StatsBase.jl)
- Tabular data (Tables.jl)
- StatsModels for (formulae / data => modeling components)
and others such as Optimization through Optim.jl and AD, etc.

Still, there are a few things I would like to move towards,
- Reusing the LSMR from FixedEffects.jl
- Finish up the predict methods for MLJ.jl
- Port some of the code to shared packages
  - CovarianceMatrices.jl
  - RegressionTables.jl
  - Some interest on a nice utility for ANOVA / LR/Wald/Score tests
Other goals include the current open issues in the repo, time series support, and streaming data.

One big challenge is having a nice API for the components to play nicely together.

As for dev path, I am happy to try a few approaches to get a feel of the best approaches.

**Comment by caseykneale** [comment]
Just eyeballing your code we have some similar elements for how we decided to align things. Also looks like we tried to get some similar features implemented. I don't have a ton of time to work on my package anymore, but, I really love how your package focusses on core analytics. 

I jumped more to machine learning type things and specialized tools assuming the end user could access those things. Not sure exactly how or if we can directly integrate but I'll be keeping it in the back of my mind.
